packages/office-addin-test-server/src/testServer.ts

```diff
@@ -48,22 +50,31 @@ export class TestServer {
                     resolveResults(this.m_jsonData);
                 });
             });
-    
-            this.m_server = https.createServer(options, this.m_app);
-    
+
             // start listening on specified port
+            return await this.startListening();
+
+        } catch (err) {
+            throw new Error(`Unable to start test server.\n${err}`);
+        }
+    }
+
+    private async startListening(): Promise<any> {
```

Comment:
This should be "Promise<boolean>" as well.

Reply:
fixed locally